alchemy/src/util/telemetry/context.ts

```diff
@@ -132,7 +132,7 @@ function environment(): Telemetry.Context["environment"] {
 
 export async function context(input: {
   sessionId: string;
-  phase: Phase;
+  phase?: Phase;
```

Comment:
Why did this become optional?

Reply:
so we can use the same telemetry client in the cli, and the cli doesn't have a phase.